rhoas/kafkas/resource_kafka.go

```diff
@@ -139,7 +143,9 @@ func kafkaDelete(ctx context.Context, d *schema.ResourceData, m interface{}) dia
 
 	_, err = deleteStateConf.WaitForStateContext(ctx)
 	if err != nil {
-		return diag.FromErr(errors.Wrapf(err, "Error waiting for example instance (%s) to be deleted", d.Id()))
+		if !strings.Contains(err.Error(), "not found") {
+			return diag.FromErr(errors.Wrapf(err, "Error waiting for example instance (%s) to be deleted", d.Id()))
+		}
```

Comment:
Maybe I am missing something here but why do we not care about the error if it contains "not found"

Reply:
I found that sometimes the API returns an error "not found" when the resource has been deleted and the terraform provider fails. So I thought that a "not found" is OK if we are deleting the resource

Reply:
Okay this is kind of weird but maybe there is no way around it.